src/main/kotlin/com/github/gradle/node/util/PlatformHelper.kt

```diff
@@ -9,9 +9,8 @@ open class PlatformHelper constructor(private val props: Properties = System.get
             name.contains("windows") -> "win"
             name.contains("mac") -> "darwin"
             name.contains("linux") -> "linux"
-            name.contains("freebsd") -> "linux"
             name.contains("sunos") -> "sunos"
-            else -> error("Unsupported OS: $name")
+            else -> "unsupported"
```

Comment:
I don't really like this since if we're extending this or someone uses their own distribution URL which *does* have binaries it's going to fail, and we're providing no way to override it.